server/game/cards/06-CotE/FireTensaiAcolyte.js

```diff
@@ -0,0 +1,14 @@
+const DrawCard = require('../../drawcard.js');
+
+class FireTensaiAcolyte extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: context => context.player.opponent && !this.game.isDuringConflict('fire'),
```

Comment:
why do you have `context.player.opponent` here?

Reply:
I thought that was required for single player games or something like that.